tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "compileOnSave": true,
+  "compilerOptions": {
+    "outDir": "dist",
+    "target": "es5",
+    "module": "commonjs",
```

Comment:
Does this produce correct output in rollup?

Reply:
yeah it seems to. I think the typescript rollup plugin takes care of a few things